xpk.py

```diff
@@ -229,6 +229,14 @@
         command: [ "sleep", "inf" ]
 """
 
+cluster_configmap_yaml = """kind: ConfigMap
+apiVersion: v1 
```

Comment:
The config map is great. I am thinking we will use the config map to store additional data. How do you recommend we do this? Like for example, we want to store a XPK version that the cluster was created with.

One way I think we could do this is we set a custom name for each config map?

Reply:
I was looking into some nesting structure but I don't think ConfigMap support that. We would have to create separate config map to store metadata like XPK version.

Reply:
Thinking a bit more, I think we can change the config map name here to align with its purpose (to store the resource types of the cluster). This will also, allow us to create additional config maps easier

Reply:
yes, will change that.